src/main/resources/application.yml

```diff
@@ -10,8 +10,8 @@ spring:
 
   datasource:
     hikari:
-      maximum-pool-size: 5
-      connectionTimeout: 1000
+      maximum-pool-size: 10
+      connectionTimeout: 2000
```

Comment:
Doubling the connection pool, total connections is limited by AWS RDS class, for us it is 198.
 25 * 4 pods is fine.
We were getting some timeouts, 2 seconds isn't unreasonably high.